packages/react-searchbox/src/components/SearchBox.js

```diff
@@ -76,7 +76,8 @@ class SearchBox extends React.Component {
     if (!isEmpty(focusShortcuts)) {
       this.hotKeyCombinationsUsed = isHotkeyCombinationUsed(focusShortcuts);
       if (this.hotKeyCombinationsUsed) {
-        import('hotkeys-js')
+        const moduleName = 'hotkeys-js';
+        import(moduleName)
```

Comment:
@mohdashraf010897 How does that fix the issue `hotkeys-js` lib not found even if the user is not using combination?

Reply:
https://www.loom.com/share/847a2f6a6f6140029152dc6a0ee8f4a7